SPECS/python-mako/python-mako.spec

```diff
@@ -1,15 +1,19 @@
 Summary:        Python templating language
 Name:           python-mako
-Version:        1.0.7
-Release:        5%{?dist}
+Version:        1.1.5
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages/Python
 URL:            https://www.makotemplates.org/
-Source0:        https://files.pythonhosted.org/packages/eb/f3/67579bb486517c0d49547f9697e36582cd19dafb5df9e687ed8e22de57fa/Mako-1.0.7.tar.gz
+Source0:        https://github.com/sqlalchemy/mako/archive/refs/tags/rel_1_1_5.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
Nit: Can we use a macro to generate the underscored version?  Something like: 

`%define version_tag %(echo %{version} | cut -d. -f1-3 --output-delimiter="-")`